src/plugins/data-card/data-card-content.ts

```diff
@@ -29,9 +28,7 @@ export function defaultDataSet() {
 }
 
 export function defaultDataCardContent(props?: IDefaultContentOptions): DataCardContentModelType {
-  const content = DataCardContentModel.create();
-  props?.title && content.setTitle(props.title);
```

Comment:
This `content.setTitle(props.title);` was actually not doing anything since this content didn't have a parent.